TF_Settings_Web/src/customHooks.ts

```diff
@@ -25,6 +27,8 @@ export const useIsFullScreen = () => {
     return isFullScreen;
 };
 
+export const useIsBrightSign = (): boolean => platform.os?.family === 'Linux' || platform.os?.family === 'Ubuntu';
```

Comment:
💬 As we're going to be doing RPi/Linux work very soon, this is likely to get misleading. Should this be called `isLinux` if there are no other BRS identifying features we could check for? Although we might just say we don't officially support TouchFree on Linux other than BRS and Raspian in which case this is fine!

Reply:
Will update to isLinux for now - want to test on BRS to see if there is any more specific information in the platform object to identify it as BRS (instead of just linux)